test/unit/logic/round.js

```diff
@@ -487,38 +448,33 @@ describe('rounds', () => {
 	describe('flushRound', () => {
 		let stub;
 		let res;
-		let scope;
 
-		before(done => {
-			scope = _.cloneDeep(validScope);
-			round = new Round(_.cloneDeep(scope), db);
+		beforeEach(async () => {
+			stub = scope.library.storage.entities.Round.delete.resolves('success');
+			round = new Round(_.cloneDeep(scope), dbStub);
 			res = round.flushRound();
-			done();
 		});
 
 		it('should return promise', () => {
 			return expect(isPromise(res)).to.be.true;
 		});
 
-		it('query should be called with proper args', () => {
-			return res.then(response => {
-				expect(response).to.equal('success');
-				stub = validScope.library.storage.entities.Round.delete;
-				expect(stub.calledWith({ round: validScope.round })).to.be.true;
-			});
+		it('query should be called with proper args', async () => {
+			const response = await res;
+			expect(response).to.equal('success');
+			expect(stub).to.be.calledWith({ round: validScope.round });
```

Comment:
There should be no checks with `validScope`, as now this variable is used only for cloning from.